Dockerfile

```
@@ -0,0 +1,11 @@
+FROM articulate/articulate-node:12-stretch-slim
+ARG NPM_TOKEN=secret
+
+RUN apt-get update -qq && apt-get install -y p7zip-full
+
+ENV npm_config_loglevel=warn
+
+COPY --chown=service:service package.json yarn.lock $SERVICE_ROOT/
+COPY --chown=service:service . $SERVICE_ROOT/
+
+RUN yarn install --pure-lockfile && yarn cache clean && npm link
```

This doesn't work for me - `yarn` seems to have removed the `--pure-lockfile` arg in version 2. I believe it's now the default not to update the lockfile on `yarn install`.

```
...
Step 7/7 : RUN yarn install --pure-lockfile && yarn cache clean && npm link
 ---> Running in 18baaa390484
Unknown Syntax Error: Unsupported option name ("--pure-lockfile").
```

```suggestion
RUN yarn install && yarn cache clean && npm link
```

---

Disregard - this was due to a local `.yarn` on my machine from running yarn outside the container. Deleting that made this work fine.